magicctapipe/scripts/lst1_magic/lst1_magic_create_irf.py

```diff
@@ -2,27 +2,30 @@
 # coding: utf-8
 
 """
-This script processes MC DL2 events and creates the IRFs. Now it can
-create only point-like IRFs, i.e., the effective area, energy migration
-and background model. If the input data is only gamma MC, it also skips
-the creation of the background model.
-
-There are 4 different IRF types which can be specified by the "irf_type"
-setting in the configuration file. The "hardware" type is supposed for
-the hardware trigger between LST-1 and MAGIC, allowing for the events of
-all the telescope combinations. The "software(_only_3tel)" types are
-supposed for the software coincidence with LST-mono and MAGIC-stereo,
+This script processes MC DL2 events and creates the IRFs. It can create
+two different IRF types based on the number of FoV offset bins,
+"POINT-LIKE" in case of one bin and "FULL-ENCLOSURE" in the other cases.
+The effective area and energy dispersion HDUs are created in case of the
+"POINT_LIKE" IRFs, and in addition the PSF table and background HDUs in
+case of the "FULL-ENCLOSURE" IRFs. When the input gamma MC is point-like
+data, it creates one bin around the true FoV offset and creates only the
+"POINT-LIKE" IRFs.
```

Comment:
Maybe you can add also here the detail that ring-wobble MCs will be treated as point-like MCs.